src/views/RoomSelector.vue

```diff
@@ -108,6 +114,16 @@ export default {
 			type: String,
 			default: '',
 		},
+
+		noMatchFoundTitle: {
```

Comment:
this could be a computed prop:
```
return this.listOpenConversations
  ? t('spreed', 'No open conversations found')
  : t('spreed', 'No conversations found')
  
  same for subtitle

Reply:
Let's try to minimize changes in this component: it is used also by external apps (deck.js, maps.js) and it could be hard to test, if we break something there or not